wasm-text.md

```diff
@@ -133,8 +213,145 @@ In the text format, it is also allowed to have a conditional without the `else`
        andBool ( ID ==K OID'' orBool notBool isIdentifier(OID'') )
 ```
 
-Memory and Tables
------------------
+Definitions
+-----------
+
+TODO: It's pretty ugly to access `NEXTIDX` the way we are doing here, ideally it should be more functional.
+
```

Comment:
Probably nicer to either put `#saveIdentifer` *before* the defintion (and avoid the `-Int 1` part), or just get rid of this and inline the identifier updates.